app/logic/Calendar/OWA/OWAEvent.ts

```diff
@@ -133,14 +133,27 @@ export class OWAEvent extends Event {
     */
   }
 
-  async saveCalendarItem() {
-    let request: any = this.itemID ?
+  getItemRequest() {
```

Comment:
I've changed this to protected, to make it clearer that this is just a helper function.